packages/server/postgres/migrations/1709312768000_scheduledJobMove.ts

```diff
@@ -0,0 +1,72 @@
+import {FirstParam} from 'parabol-client/types/generics'
+import {Client} from 'pg'
+import {r} from 'rethinkdb-ts'
+import getPgConfig from '../getPgConfig'
+import connectRethinkDB from '../../database/connectRethinkDB'
+import getPgp from '../getPgp'
+import {ScheduledJob as PgScheduledJob} from '../pg'
+
+export async function up() {
+  await connectRethinkDB()
+  const {pgp, pg} = getPgp()
+  const batchSize = 1000
+
+  const columnSet = new pgp.helpers.ColumnSet(
+    ['runAt', 'type', {name: 'orgId', def: null}, {name: 'meetingId', def: null}],
+    {table: 'ScheduledJob'}
+  )
+
+  const transformRethinkRow = (row: any): Omit<PgScheduledJob, 'id'> => {
+    const {id, runAt, type, orgId, meetingId} = row
+    return {
+      runAt,
+      type,
+      orgId,
+      meetingId
+    }
+  }
+
+  const getNextData = async (leftBoundCursor: Date | undefined) => {
```

Comment:
Note: see https://github.com/ParabolInc/parabol/blob/ba7d7246c8db632f8016ec632c7841ae9a847dd2/packages/server/postgres/migrations/1677272969994_meetingTemplatesMove.ts#L37 for where I pilfered this logic